src/platform/iotjs_module_i2c-linux-general.inl.h

```diff
@@ -212,6 +259,61 @@ void OpenWorker(uv_work_t* work_req) {
 }
 
 
+void ScanWorker(uv_work_t* work_req) {
+  I2cReqWrap* i2c_req = reinterpret_cast<I2cReqWrap*>(work_req->data);
+  I2cReqData* req_data = i2c_req->req();
+
+  int result;
+  req_data->buf_data = AllocBuffer(128);
+  req_data->buf_len = 128;
+
+  for (int i = 0; i < 128; i++) {
+    ioctl(fd, I2C_SLAVE_FORCE, i);
+
+    if ((i >= 0x30 && i <= 0x37) || (i >= 0x50 && i <= 0x5F)) {
```

Comment:
I think '128' is good to be maintained as literals however 0x30, 0x37, 0x50, 0x5F is originated from other I2C related code. It would be enough to add comments which describes the meaning of these values.